modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -39,16 +39,30 @@ export default class NestedTabs extends Base {
 		return tabTitleElement.getAttribute( 'data-tab-index' );
 	}
 
+	getActiveTabIndex() {
+		const settings = this.getSettings(),
+			activeTitleFilter = settings.ariaAttributes.activeTitleSelector,
+			$activeTitle = this.elements.$tabTitles.filter( activeTitleFilter );
+
+		return $activeTitle.attr( 'data-tab-index' ) || null;
+	}
+
+	getWidgetNumber() {
+		return this.$element.find( '> .elementor-widget-container > .e-n-tabs, > .e-n-tabs' ).attr( 'data-widget-number' );
```

Comment:
`> .elementor-widget-container > .e-n-tabs, > .e-n-tabs` can be added inside `getDefaultSettings()`
Also `data-widget-number`

Reply:
The problem is that I call this from the `getDefaultSettings()`. I fear that it won't work in that way.

Reply:
is `getWidgetNumber` running before `getDefaultSettings`?
I'll be glad if we can try that.

Reply:
I call it from the getDefaultSettings() on line 56.